R/lookup.R

```diff
@@ -2,43 +2,133 @@
 # practical purpose.
 investor_name_placeholder <- "Meta Investor"
 
+#' Help lookup values
```

Comment:
I am wondering if we should not better restrict the export to vars that are advertised in the documentation of run_stress_test, e.g. lgd_senior_claims_range_lookup and keep the remaining lookups as internal use only, as they should not be of relevance to the users.

Reply:
Noted. TODO.

Reply:
https://github.com/2DegreesInvesting/r2dii.climate.stress.test/pull/143